pkg/export/setup/setup.go

```diff
@@ -129,6 +129,15 @@ func FromFlags(a *kingpin.Application, userAgent string) func(log.Logger, promet
 	a.Flag("export.debug.batch-size", "Maximum number of points to send in one batch to the GCM API.").
 		Default(strconv.Itoa(export.BatchSizeMax)).UintVar(&opts.BatchSize)
 
+	a.Flag("export.token-url", "The request URL to generate token that's needed to ingest metrics to an alternative project").
+		StringVar(&opts.TokenURL)
+
+	a.Flag("export.token-body", "The request Body to generate token that's needed to ingest metrics to an alternative project.").
```

Comment:
request body

Can you elaborate on the "alternative project" bit?

Reply:
This bit is incorrect. Updated the text to "to generate token that's needed to ingest metrics to the project".